content/03.results.md

```diff
@@ -38,7 +38,9 @@ of the containment index:
 ```
 {% endraw %}
 
-See Methods for details.
+See Methods for details. Detailed analysis of bias and confidence
```

Comment:
I think this sentence is not in the best place. Perhaps one paragraph down, after "This permits a variety of convenient operations on the sketches, including iterative downsampling of FracMinHash sketches as well as conversion to MinHash sketches." 
One could add 
"Beyond accurate containment operations, FracMinHash can be used to estimate evolutionary distance between pairs of sequences undergoing a mutation model, similar to but more accurately than the MinHash derived method in [@mash]. See [@dk_fracminhash] for these and other analytical details."

Reply:
updated in 56039295ecffb3bd6485efe099530bfc7393eeea